content/filters/_index.md

```diff
@@ -18,13 +18,14 @@ We have already defined a filter to add suggestions in the notifications if cont
 Prerequisites:
 
 - As of now, you can write filters only using Go language. So you need to be familiar with it.
+- Understanding of Kubernetes Objects needed (https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/)
 
 #### 1. Create a new .go file
 Create a new file (e.g image_tag_checker.go) in **botkube/pkg/filterengine/filters/** directory
 
 Set package name as "filters" and import required packages
 
-```go
+```

```

Comment:
`go` removed ?

Reply:
It wasn't working anyways, so removed it